src/file.spec.ts

```diff
@@ -1,4 +1,7 @@
-import {COMMON_MIME_TYPES, toFileWithPath} from './file';
+import {COMMON_MIME_TYPES, toFileWithPath, clone} from './file';
+import {MockFile} from './file.mock';
+
+declare let global: any;
```

Comment:
`global` should already exist as far as I remember (on the node namespace). If it causes you problems when you use it, just redeclare as `const g = global as any` in your test.

Reply:
Will add that.